jaeger/http_client.go

```diff
@@ -53,6 +53,13 @@ func getTraceDetailHTTP(client http.Client, endpoint *url.URL, traceID string) (
 }
 
 func queryTracesHTTP(client http.Client, u *url.URL) (*JaegerResponse, error) {
+	// HTTP and GRPC requests co-exist, but when minDuration is present, for HTTP it requires a unit (ms)
+	// https://github.com/kiali/kiali/issues/3939
+	if u.Query().Get("minDuration") != "" {
```

Comment:
For the safe side, isn't it better to check strings.HasSuffix(u.Query().Get("minDuration"), "ms") ?

Reply:
Right ! Added a fix